session/bootstrap.go

```diff
@@ -2721,6 +2729,18 @@ func upgradeToVer173(s Session, ver int64) {
 	doReentrantDDL(s, "ALTER TABLE mysql.tidb_background_subtask ADD COLUMN `summary` JSON", infoschema.ErrColumnExists)
 }
 
+func upgradeToVer174(s Session, ver int64) {
+	if ver >= version174 {
+		return
+	}
+	writeStatsGCLastPos(s)
+}
+
+func writeStatsGCLastPos(s Session) {
+	mustExecute(s, "INSERT HIGH_PRIORITY INTO %n.%n VALUES(%?, %?, %?) ON DUPLICATE KEY UPDATE VARIABLE_VALUE = %?",
+		mysql.SystemDB, mysql.TiDBTable, tiDBStatsGCLastTS, 0, tiDBStatsGCLastTSComment, 0)
+}
```

Comment:
I think we don't need this actually.
We can just treat the empty result as `0` when reading this record, and use `insert ... on duplicate key update` when writing this record.